### PR TITLE
Search for forms and faq articles on home page

### PR DESCRIPTION
**Pull request by AdrienClairembault**
## Checklist before requesting a review

- [X] I have performed a self-review of my code.
- [X] I have added tests that prove my fix is effective or that my feature works.

## Description

Allow user to search for forms and faq articles on home page.

## Screenshots (if appropriate):

image

### CODE REVIEW

File: templates/pages/helpdesk/search.html.twig [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                    class="row align-items-center"
+                    aria-label="{{ faq.name }}"
+                >
+                    <div class="col-auto">
+                        {{ render_illustration("browse-help.svg", 60) }}
+                    </div>
+                    <div class="col text-truncate">
+                        <span class="text-dark text-reset d-block">{{ faq.name }}</span>
+                        <div class="d-block text-secondary text-truncate richtext-strip-margin">
+                            {{ faq.answer|raw }}
```
Comment (cedric-anne) [inline templates/pages/helpdesk/search.html.twig:53]:
```suggestion
                            {{ faq.answer|safe_html }}
```

**Review by cedric-anne — CHANGES_REQUESTED** [review]

**Review by cedric-anne — APPROVED** [review]

**Comment by orthagh** [comment]
A question also, how behave the description if (very) long ?

**Comment by AdrienClairembault** [comment]
> What will happen if there is 50 FAQ entries and 10 forms matching the searched term ?

I want to add some kind of infinite scroll (or similar solutions) later.
For now we can put an hardcoded limit to 20.

**Comment by AdrienClairembault** [comment]
> A question also, how behave the description if (very) long ?

There is an ellipsis:

image

**Review by orthagh — APPROVED** [review]
Please add the limit, yes. Anyway OK for me.

**Review by cedric-anne — APPROVED** [review]

**Pull request merged**